src/renderer/tvgPaint.cpp

```diff
@@ -88,13 +123,15 @@ static Result _compFastTrack(Paint* cmpTarget, const RenderTransform* pTransform
         if (v1.x > v2.x) std::swap(v1.x, v2.x);
         if (v1.y > v2.y) std::swap(v1.y, v2.y);
 
-        viewport.x = static_cast<int32_t>(v1.x);
-        viewport.y = static_cast<int32_t>(v1.y);
-        viewport.w = static_cast<int32_t>(ceil(v2.x - viewport.x));
-        viewport.h = static_cast<int32_t>(ceil(v2.y - viewport.y));
+        after.x = static_cast<int32_t>(v1.x);
+        after.y = static_cast<int32_t>(v1.y);
+        after.w = static_cast<int32_t>(ceil(v2.x - after.x));
+        after.h = static_cast<int32_t>(ceil(v2.y - after.y));
+
+        if (after.w < 0) after.w = 0;
+        if (after.h < 0) after.h = 0;
```

Comment:
in case after.w or after.h <= 0 there is no point of intersecting the regions, right? I think we could set w=h=0 and return

Reply:
It's ignorable. No point for a super corner case. Let's keep the code clean.

Reply:
ok, so no comments otherwise